### PR TITLE
Website

### PR DESCRIPTION
**Pull request by Divinity-dev**
In this branch of the project, I implemented the following functions.
1. I created a Navbar to navigate through the page.
2.I created 3 pages(Home, calculator and Quotes).
3. I used routers to implement navigation
4. I added little css styling.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
   return (
-    <Calculator />
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/Home" element={<Home />} />
+        <Route path="/Calculator" element={<Calculator />} />
+        <Route path="/Quotes" element={<Quotes />} />
+      </Routes>
+    </>
```
Comment (iambenkis) [inline src/App.js:17]: - [ ] Keep doing the great work :+1:

**Review by iambenkis — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @Divinity-dev  :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Basic Structure is good
:heavy_check_mark: Can be maximized upto 200%


Good job on implementing some of the requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Required changes :recycle: 

- [ ] Good job setting up your README file, but it could be better if you remove the `logo` because it is a broken link :+1:  to make your file more professional :100: 
Screenshot from 2023-03-01 01-26-48

- [ ] Kindly make sure that you select only the stack you used else you should use `N/A`
Screenshot from 2023-03-01 01-27-33

- [ ] Kindly make sure you provide a title to your project more professionalism
Screenshot from 2023-03-01 01-27-10

- [ ] Kindly remove this not because they are not useful :+1: 
Screenshot from 2023-03-01 01-28-02


# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @iambenkis in your question, so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]
Hi @Divinity-dev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️ Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
Thank you 

Sent from Yahoo Mail on Android 
 
  On Wed, Mar 1, 2023 at 12:58 AM, ***@***.***> wrote:   


@alexander16108 approved this pull request.

Hi @Divinity-dev ,

Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉
 


Highlights

 ✔️ No linter errors
 ✔️ Gitflow was correctly used
 ✔️ Good PR and Description

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**